domino/domino.py

```diff
@@ -136,16 +136,18 @@ def runs_start_blocking(self, command, isDirect=False, commitId=None,
         run_response = self.runs_start(command, isDirect, commitId, title,
                                        tier, publishApiEndpoint)
         run_id = run_response['runId']
-
+        
         poll_start = time.time()
         current_retry_count = 0
         while True:
             try:
                 run_info = self.get_run_info(run_id)
+                if run_info is None:
+                    raise RunNotFoundException(f"Tried to access nonexistent run id {run_id}")           
```

Comment:
Please add an import statement at the top
```
from .exceptions import RunNotFoundException
```

Reply:
Done